### PR TITLE
hledger 0.27 (new formula)

### PR DESCRIPTION
**Pull request by travisbhartwell**
Add formula for hledger accounting tool, the command-line version.

Added since the discussion in #4430 determined this was now viable.

### CODE REVIEW

File: Library/Formula/hledger.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+class Hledger < Formula
+  include Language::Haskell::Cabal
+
+  desc "The hledger accounting tool, command-line version"
+  homepage "http://hledger.org"
+  url "http://hackage.haskell.org/package/hledger-0.27/hledger-0.27.tar.gz"
+  sha256 "0aecdf586a46c24d6d67659157d1edbfc0d78afb50ea7cfbec1a01bf86b792b0"
+
+  # head "https://github.com/simonmichael/hledger.git"
```
Comment (DomT4) [inline Library/Formula/hledger.rb]: Please either include this or cut the line :wink:

File: Library/Formula/hledger.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+  bottle do
```
Comment (DomT4) [inline Library/Formula/hledger.rb]: Don't need this block. We add bottles ourselves on pull.

File: Library/Formula/hledger.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
+require "language/haskell"
+
+class Hledger < Formula
+  include Language::Haskell::Cabal
+
+  desc "The hledger accounting tool, command-line version"
+  homepage "http://hledger.org"
+  url "http://hackage.haskell.org/package/hledger-0.27/hledger-0.27.tar.gz"
```
Comment (DomT4) [inline Library/Formula/hledger.rb]: This should be `https://` I believe.

File: Library/Formula/hledger.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
+require "language/haskell"
+
+class Hledger < Formula
+  include Language::Haskell::Cabal
+
+  desc "The hledger accounting tool, command-line version"
```
Comment (DomT4) [inline Library/Formula/hledger.rb]: Ideally don't want the name in the `desc`; we auto-print it.

**Comment by travisbhartwell** [comment]
@DomT4 Thanks for the feedback.  I'll update my commit and do a force push tomorrow to address it.

**Comment by travisbhartwell** [comment]
@DomT4 Hopefully I've addressed the issues with my latest push.

File: Library/Formula/hledger.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
+require "language/haskell"
+
+class Hledger < Formula
+  include Language::Haskell::Cabal
+
+  desc "The command-line accounting tool"
```
Comment (bfontaine) [inline Library/Formula/hledger.rb]: You could use `"Command-line accounting tool"`.

**Comment by travisbhartwell** [comment]
@bfontaine Updated.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/5ae82bcc5b02e61e1fda2b77a625787d47aa8cfb. Thank you for your contribution to Homebrew @travisbhartwell; we appreciate it! :smiley_cat:

**Pull request closed without merging**